be/src/storage/rowset_merger.cpp

```diff
@@ -260,7 +260,9 @@ class RowsetMergerImpl : public RowsetMerger {
         std::vector<ColumnId> sort_key_idxes;
         if (schema.sort_key_idxes().size() > 1) {
             sort_key_idxes = schema.sort_key_idxes();
-            if (!PrimaryKeyEncoder::create_column(schema, &sort_column, sort_key_idxes).ok()) {
+            std::vector<ColumnId> sort_key_iota_idxes(sort_key_idxes.size());
+            std::iota(sort_key_iota_idxes.begin(), sort_key_iota_idxes.end(), 0);
```

Comment:
why use idxes rather than  sort_key_dxes? I don't see the logic here

Reply:
refactored.